app/src/test/java/org/wikipedia/connectivity/NetworkConnectivityReceiverTest.java

```diff
@@ -0,0 +1,27 @@
+package org.wikipedia.connectivity;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+import org.wikipedia.WikipediaApp;
+import org.wikipedia.analytics.eventplatform.EventPlatformClient;
+
+import static org.mockito.Mockito.anyBoolean;
+import static org.mockito.Mockito.mock;
+
+@RunWith(RobolectricTestRunner.class)
+public class NetworkConnectivityReceiverTest {
+
+    @Test
+    public void testUpdatesEventPlatformClientEnabledStateOnUpdateOnlineState() {
```

Comment:
What is this intended to test? I'm not seeing any kind of assertion or comparison.